bundles/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/command/SatelCommandBase.java

```diff
@@ -41,10 +41,10 @@
      * Creates new command basing on command code and extended command flag.
      *
      * @param commandCode
-     *            command code
+     *                        command code
```

Comment:
Please check the formatting. Usually parameter and description should be on one line.

Reply:
Done.